client/plots/matrix.interactivity.js

```diff
@@ -105,7 +95,7 @@ export function setInteractivity(self) {
 					.append('div')
 					.attr('class', 'sja_menuoption sja_sharp_border')
 					.text(k)
-					.on('click', (event) => {
+					.on('click', event => {
```

Comment:
I think `const pane = newpane(); ... /* use pane.body as holder */` should be used, with a close button instead of a menu.

Reply:
![Screenshot 2023-06-21 at 12 34 56 AM](https://github.com/stjude/proteinpaint/assets/411031/89a1ad9e-5582-499d-9899-ed98bd5c38d8)